components/kubearchive/base/kustomization.yaml

```
@@ -122,6 +122,9 @@ patches:
     metadata:
       name: kubearchive-operator
       namespace: kubearchive
+      annotations: ignore-check.kube-linter.io/unset-memory-requirements: >
+        "This is temporary to troubleshoot an issue"
+
```

sorry, suggested code was not right, it should be:
```suggestion
      annotations:
        ignore-check.kube-linter.io/unset-memory-requirements: >
          "This is temporary to troubleshoot an issue"
```